backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -378,6 +378,51 @@ public static String parseState(String s) {
     throw IllegalGraphqlArgumentException.invalidInput(s, "state");
   }
 
+  private static final Map<String, String> RESPIRATORY_SYMPTOMS =
```

Comment:
⚠️ separated out the symptoms in the two categories like on the [frontend](https://github.com/CDCgov/prime-simplereport/blob/0fc7b5bf4abfaa09e1bc63ef13e7c5aa8080c915/frontend/src/patientApp/timeOfTest/constants.ts#L23) -- thought this would make things easier to read and consistent with the frontend

Let me know if you all prefer one map of these symptoms